{"@t":"2020-09-19T18:59:07.1647270Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ServerInstProcess] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":108,"AppDomainId":6,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T18:59:07.1647270Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[KeepAlive] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":72,"AppDomainId":6,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T18:59:07.1647270Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ScheduledPublishing] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":74,"AppDomainId":6,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T18:59:07.1647270Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[TempFileCleanup] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":171,"AppDomainId":6,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T18:59:07.1647270Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[RebuildIndexesOnStartup] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":168,"AppDomainId":6,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T18:59:07.1647270Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[LogScrubber] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":108,"AppDomainId":6,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T18:59:07.1647270Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[LogScrubber] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":76,"AppDomainId":6,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T18:59:07.1836773Z","@mt":"Application shutdown. Details: {ShutdownReason}\r\n\r\n_shutDownMessage={ShutdownMessage}\r\n\r\n_shutDownStack={ShutdownStack}","ShutdownReason":"ConfigurationChange","ShutdownMessage":"CONFIG change\r\nHostingEnvironment initiated shutdown\r\nCONFIG change\r\nHostingEnvironment orsakade avslut","ShutdownStack":"   vid System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)\r\n   vid System.Environment.get_StackTrace()\r\n   vid System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()\r\n   vid System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()\r\n   vid System.Web.HttpRuntime.ShutdownAppDomain(String stackTrace)\r\n   vid System.Web.Configuration.HttpConfigurationSystem.OnConfigurationChanged(Object sender, InternalConfigEventArgs e)\r\n   vid System.Configuration.Internal.InternalConfigRoot.OnConfigChanged(InternalConfigEventArgs e)\r\n   vid System.Configuration.BaseConfigurationRecord.OnStreamChanged(String streamname)\r\n   vid System.Web.Configuration.WebConfigurationHostFileChange.OnFileChanged(Object sender, FileChangeEvent e)\r\n   vid System.Web.DirectoryMonitor.FireNotifications()\r\n   vid System.Web.Util.WorkItem.CallCallbackWithAssert(WorkItemCallback callback)\r\n   vid System.Web.Util.WorkItem.OnQueueUserWorkItemCompletion(Object state)\r\n   vid System.Threading.QueueUserWorkItemCallback.WaitCallback_Context(Object state)\r\n   vid System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)\r\n   vid System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)\r\n   vid System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()\r\n   vid System.Threading.ThreadPoolWorkQueue.Dispatch()\r\n   vid System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()","SourceContext":"Umbraco.Web.UmbracoApplicationBase","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":172,"AppDomainId":6,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T18:59:09.4635537Z","@mt":"Registered with MainDom, localContentDbExists? {LocalContentDbExists}, localMediaDbExists? {LocalMediaDbExists}","LocalContentDbExists":true,"LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:09.4645510Z","@mt":"Creating the content store, localContentDbExists? {LocalContentDbExists}","LocalContentDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:09.4665481Z","@mt":"Creating the media store, localMediaDbExists? {LocalMediaDbExists}","LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:09.7827134Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading content from local cache file","TimingId":"a7d949c","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:09.8111439Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":28,"TimingId":"a7d949c","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:09.8231120Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading media from local cache file","TimingId":"65f31da","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:09.8231120Z","@mt":"Tried to load media from the local cache file but it was empty.","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:09.8231120Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":0,"TimingId":"65f31da","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:09.8241239Z","@mt":"Loading media from local db raised warnings, will reload from database.","@l":"Warning","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"WARN ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:09.8261052Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading media from database","TimingId":"ce91d0b","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:09.9502932Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":124,"TimingId":"ce91d0b","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:10.1148436Z","@mt":"Profiler is VoidProfiler, not profiling (must run debug mode to profile).","SourceContext":"Umbraco.Web.Logging.WebProfilerComponent","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:10.2290606Z","@mt":"Initializing Umbraco internal event handlers for cache refreshing.","SourceContext":"Umbraco.Web.Cache.DistributedCacheBinderComponent","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:10.2789287Z","@mt":"Adding examine event handlers for {RegisteredIndexers} index providers.","RegisteredIndexers":3,"SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:10.3058550Z","@mt":"Starting initialize async background thread.","SourceContext":"Umbraco.Web.Search.BackgroundIndexRebuilder","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:10.3158308Z","@mt":"Starting '{MigrationName}'...","MigrationName":"UmbracoForms","SourceContext":"Umbraco.Core.Migrations.MigrationPlan","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:10.3158308Z","@mt":"At {OrigState}","OrigState":"{forms-init-complete}","SourceContext":"Umbraco.Core.Migrations.MigrationPlan","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:10.3168276Z","@mt":"Done (pending scope completion).","SourceContext":"Umbraco.Core.Migrations.MigrationPlan","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:10.3178228Z","@mt":"Initializing Umbraco Forms Events to trigger Cache Refreshers","SourceContext":"Umbraco.Forms.Core.Components.UmbracoFormsComposer","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:10.3208147Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Booted.","Duration":3709,"TimingId":"c9c699d","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"6dfb7ebc-3874-471f-aa27-126b74ac983d"}
{"@t":"2020-09-19T18:59:10.4863934Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":154,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"6b06e05d-8538-45a3-854b-edc01b78de07"}
{"@t":"2020-09-19T18:59:10.4873910Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":154,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"6b06e05d-8538-45a3-854b-edc01b78de07"}
{"@t":"2020-09-19T18:59:15.4711341Z","@mt":"Exception ({InstanceId}).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): Invalid object name 'UFForms'.\r\n   vid System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   vid System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   vid System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   vid System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   vid System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()\r\n   vid System.Data.SqlClient.SqlDataReader.get_MetaData()\r\n   vid System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)\r\n   vid System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)\r\n   vid System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)\r\n   vid System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)\r\n   vid System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)\r\n   vid System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)\r\n   vid System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)\r\n   vid StackExchange.Profiling.Data.ProfiledDbCommand.ExecuteDbDataReader(CommandBehavior behavior) i C:\\projects\\dotnet\\src\\MiniProfiler.Shared\\Data\\ProfiledDbCommand.cs:rad 208\r\n   vid System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)\r\n   vid Umbraco.Core.Persistence.FaultHandling.FaultHandlingDbCommand.<>c__DisplayClass30_0.<ExecuteDbDataReader>b__0() i D:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\FaultHandling\\RetryDbConnection.cs:rad 204\r\n   vid Umbraco.Core.Persistence.FaultHandling.FaultHandlingDbCommand.<>c__DisplayClass33_0`1.<Execute>b__0() i D:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\FaultHandling\\RetryDbConnection.cs:rad 222\r\n   vid Umbraco.Core.Persistence.FaultHandling.RetryPolicy.ExecuteAction[TResult](Func`1 func) i D:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\FaultHandling\\RetryPolicy.cs:rad 172\r\n   vid Umbraco.Core.Persistence.FaultHandling.FaultHandlingDbCommand.Execute[T](Func`1 f) i D:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\FaultHandling\\RetryDbConnection.cs:rad 219\r\n   vid Umbraco.Core.Persistence.FaultHandling.FaultHandlingDbCommand.ExecuteDbDataReader(CommandBehavior behavior) i D:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\FaultHandling\\RetryDbConnection.cs:rad 204\r\n   vid System.Data.Common.DbCommand.ExecuteReader()\r\n   vid NPoco.Database.ExecuteReaderHelper(DbCommand cmd)\r\n   vid NPoco.Database.ExecuteDataReader(DbCommand cmd)\r\nClientConnectionId:7edc0b2e-7425-40b1-aee3-80e36318f084\r\nError Number:208,State:1,Class:16","InstanceId":"140cb2e0","SourceContext":"Umbraco.Core.Persistence.UmbracoDatabase","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":65,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T18:59:15.4850698Z","@mt":"Index populating failed for populator {Populator}","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): Invalid object name 'UFForms'.\r\n   vid System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   vid System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   vid System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   vid System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   vid System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()\r\n   vid System.Data.SqlClient.SqlDataReader.get_MetaData()\r\n   vid System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)\r\n   vid System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)\r\n   vid System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)\r\n   vid System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)\r\n   vid System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)\r\n   vid System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)\r\n   vid System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)\r\n   vid StackExchange.Profiling.Data.ProfiledDbCommand.ExecuteDbDataReader(CommandBehavior behavior) i C:\\projects\\dotnet\\src\\MiniProfiler.Shared\\Data\\ProfiledDbCommand.cs:rad 208\r\n   vid System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)\r\n   vid Umbraco.Core.Persistence.FaultHandling.FaultHandlingDbCommand.<>c__DisplayClass30_0.<ExecuteDbDataReader>b__0() i D:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\FaultHandling\\RetryDbConnection.cs:rad 204\r\n   vid Umbraco.Core.Persistence.FaultHandling.FaultHandlingDbCommand.<>c__DisplayClass33_0`1.<Execute>b__0() i D:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\FaultHandling\\RetryDbConnection.cs:rad 222\r\n   vid Umbraco.Core.Persistence.FaultHandling.RetryPolicy.ExecuteAction[TResult](Func`1 func) i D:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\FaultHandling\\RetryPolicy.cs:rad 172\r\n   vid Umbraco.Core.Persistence.FaultHandling.FaultHandlingDbCommand.Execute[T](Func`1 f) i D:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\FaultHandling\\RetryDbConnection.cs:rad 219\r\n   vid Umbraco.Core.Persistence.FaultHandling.FaultHandlingDbCommand.ExecuteDbDataReader(CommandBehavior behavior) i D:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\FaultHandling\\RetryDbConnection.cs:rad 204\r\n   vid System.Data.Common.DbCommand.ExecuteReader()\r\n   vid NPoco.Database.ExecuteReaderHelper(DbCommand cmd)\r\n   vid NPoco.Database.ExecuteDataReader(DbCommand cmd)\r\n   vid NPoco.Database.<QueryImp>d__164`1.MoveNext()\r\n   vid System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)\r\n   vid System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)\r\n   vid NPoco.Database.Fetch[T](Sql sql)\r\n   vid Umbraco.Forms.Core.Persistence.Repositories.Implement.FormRepository.PerformGetAll(Guid[] ids)\r\n   vid Umbraco.Core.Cache.NoCacheRepositoryCachePolicy`2.GetAll(TId[] ids, Func`2 performGetAll) i D:\\a\\1\\s\\src\\Umbraco.Core\\Cache\\NoCacheRepositoryCachePolicy.cs:rad 47\r\n   vid Umbraco.Core.Persistence.Repositories.Implement.RepositoryBase`2.GetMany(TId[] ids) i D:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\Repositories\\Implement\\RepositoryBaseOfTIdTEntity.cs:rad 192\r\n   vid Umbraco.Forms.Core.Services.BaseService`4.Get()\r\n   vid Umbraco.Forms.Core.Data.RecordIndex.FormsIndexPopulator.PopulateIndexes(IReadOnlyList`1 indexes)\r\n   vid Umbraco.Examine.IndexPopulator.Populate(IIndex[] indexes) i D:\\a\\1\\s\\src\\Umbraco.Examine\\IndexPopulator.cs:rad 41\r\n   vid Umbraco.Examine.IndexRebuilder.RebuildIndexes(Boolean onlyEmptyIndexes) i D:\\a\\1\\s\\src\\Umbraco.Examine\\IndexRebuilder.cs:rad 68\r\nClientConnectionId:7edc0b2e-7425-40b1-aee3-80e36318f084\r\nError Number:208,State:1,Class:16","Populator":"Umbraco.Forms.Core.Data.RecordIndex.FormsIndexPopulator","SourceContext":"Umbraco.Examine.IndexRebuilder","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":65,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T19:02:16.4674488Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":38,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T19:02:36.6375751Z","@mt":"Scheduled health check results:","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6375751Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Folder & File Permissions","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6375751Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"All folders have the correct permissions set.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6375751Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"All files have the correct permissions set.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"SMTP Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"WARN "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"SMTP Settings","HealthCheckResult":"Error","HealthCheckMessage":"In the Web.config file system.net/mailsettings section, the host is not configured.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Macro errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"WARN "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Macro errors","HealthCheckResult":"Error","HealthCheckMessage":"MacroErrors are set to 'throw' which will prevent some or all pages in your site from loading completely if there are any errors in macros. Rectifying this will set the value to 'inline'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Notification Email Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"WARN "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Notification Email Settings","HealthCheckResult":"Error","HealthCheckMessage":"Notification email is still set to the default value of <strong>your@email.here</strong>.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Try Skip IIS Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Try Skip IIS Custom Errors","HealthCheckResult":"Success","HealthCheckMessage":"Try Skip IIS Custom Errors is set to 'true' and you're using IIS version '10.0'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Custom Errors","HealthCheckResult":"Success","HealthCheckMessage":"Custom errors are set to 'RemoteOnly'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Trace Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Trace Mode","HealthCheckResult":"Success","HealthCheckMessage":"Trace mode is disabled.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Debug Compilation Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Debug Compilation Mode","HealthCheckResult":"Success","HealthCheckMessage":"Debug compilation mode is disabled.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"WARN "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Det går inte att ansluta till fjärrservern'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"WARN "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Det går inte att ansluta till fjärrservern'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Content/MIME Sniffing Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"WARN "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Content/MIME Sniffing Protection","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Det går inte att ansluta till fjärrservern'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Excessive Headers","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"WARN "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Excessive Headers","HealthCheckResult":"Warning","HealthCheckMessage":"[httpsCheckInvalidUrl]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Click-Jacking Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"WARN "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Click-Jacking Protection","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Det går inte att ansluta till fjärrservern'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"HTTPS Configuration","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"WARN "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Error","HealthCheckMessage":"You are currently not viewing the site using the HTTPS scheme.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Info","HealthCheckMessage":"The appSetting 'Umbraco.Core.UseHttps' is set to 'false' in your web.config file. Once you access this site using the HTTPS scheme, that should be set to 'true'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL https://localhost/umbraco - 'Det går inte att ansluta till fjärrservern'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Database data integrity check","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database data integrity check","HealthCheckResult":"Success","HealthCheckMessage":"<p>All document paths are valid</p>\r\n","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:02:36.6385716Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database data integrity check","HealthCheckResult":"Success","HealthCheckMessage":"<p>All media paths are valid</p>\r\n","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"INFO "}
{"@t":"2020-09-19T19:07:20.4831907Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T19:12:24.4991869Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T19:17:28.5258030Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":13,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T19:22:32.5523455Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":13,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T19:27:36.5748399Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":158,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T19:32:40.5970802Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":158,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T19:37:44.6198900Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":13,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T19:42:48.6407897Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":158,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T19:47:52.8627629Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":158,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T19:52:56.8841504Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T19:58:01.3597519Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T20:03:05.3791107Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T20:08:09.4008262Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T20:13:13.4195208Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T20:18:17.4402187Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T20:23:21.4638580Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T20:28:25.4878102Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T20:33:29.5139960Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":158,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T20:38:33.5358942Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":158,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T20:43:37.5608272Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":158,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T20:48:41.5758480Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":158,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T20:53:45.6007066Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T20:58:49.6239023Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T21:03:53.6394485Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T21:08:57.6561861Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":158,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T21:14:01.6769211Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T21:19:05.6849097Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T21:24:09.7033196Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":158,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T21:29:13.7201387Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T21:34:17.7436535Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T21:39:21.7508948Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":158,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T21:44:25.7729055Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":124,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T21:49:29.7796750Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":124,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T21:54:33.8033424Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":19,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
{"@t":"2020-09-19T21:59:37.8115272Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":8964,"ProcessName":"iisexpress","ThreadId":124,"AppDomainId":7,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"MSI","Log4NetLevel":"ERROR"}
